ecosystem/indexer-grpc/indexer-grpc-table-info/src/runtime.rs

```diff
@@ -66,26 +67,36 @@ pub fn bootstrap(
     let db =
         open_db(db_path, &rocksdb_config).expect("Failed to open up indexer async v2 db initially");
 
+    let table_info_gcs_bucket_name = node_config.indexer_table_info.gcs_bucket_name.clone();
     let indexer_async_v2 =
         Arc::new(IndexerAsyncV2::new(db).expect("Failed to initialize indexer async v2"));
     let indexer_async_v2_clone = Arc::clone(&indexer_async_v2);
 
     // Spawn the runtime for table info parsing
     runtime.spawn(async move {
+        let backup_restore_operator: Arc<GcsBackupRestoreOperator> =
+            Arc::new(GcsBackupRestoreOperator::new(table_info_gcs_bucket_name).await);
         let context = Arc::new(Context::new(
             chain_id,
             db_rw.reader.clone(),
             mp_sender,
             node_config.clone(),
             None,
         ));
+        // DB backup is optional
+        let backup_restore_operator = if node_config.indexer_table_info.db_backup_enabled {
```

Comment:
Should we new the GcsBackupRestoreOperator here only when it is enabled?

Reply:
done.